src/pages/EmployeeUpdate/Form.tsx

```diff
@@ -0,0 +1,394 @@
+          name: "skills",
+          placeholder: "Select skills",
+          isMulti: true,
+          isRequired: true,
+        },
+      ],
+    },
+    {
+      sectionName: "Uploads",
+      sectionActiveState: 3,
+      sectionFields: [
+        {
+          label: "Employee Photograph",
+          type: "file",
+          name: "photoId",
+          accept: "image/*",
+          isRequired: true,
+        },
+      ],
+    },
+  ];
+  return (
+    <>
+      <span
+        className="material-symbols-outlined back-btn"
+        onClick={() => navigate(-1)}
+      >
+        reply
+      </span>
+      <FormWrapper>
+        <h2>
+          {employeeId
+            ? `Edit Employee: ${
+                employeeData?.firstName + " " + employeeData?.lastName
+              }`
+            : "Add New Employee"}
+        </h2>
+        <ProgressBar
+          activeSection={activeSection}
+          steps={[
+            "Personal Details",
+            "Professional Details",
+            "Uploads",
+            "Review",
+            "Submit",
+          ]}
+        ></ProgressBar>
+        <FormProvider {...methods}>
+          <form onSubmit={(e) => e.preventDefault()} noValidate>
+            {activeSection<=3 && [formConfig
+              .find(
+                (formSection) =>
+                  activeSection === formSection.sectionActiveState
+              )!]
+              .map((formSection) => {
+                return (
+                  <React.Fragment key={formSection.sectionActiveState}>
+                    {activeSection === formSection.sectionActiveState && (
+                      <Fieldset
+                        key={formSection.sectionActiveState}
+                        className="form-details "
+                      >
+                        <h2 className="form-section-heading">
+                          {formSection.sectionName}
+                        </h2>
+                        <>
+                          {formSection.sectionFields.map(
+                            (sectionField: IInputProps) => (
+                              <Input
+                                key={sectionField.name}
+                                config={sectionField}
+                              />
+                            )
+                          )}
+                        </>
+                      </Fieldset>
+                    )}
+                  </React.Fragment>
+                );
+              })}
+
+            {activeSection === 4 && (
+              <>
+                {" "}
+                <Fieldset className="form-details ">
+                  <h2 className="form-section-heading">Review</h2>
+                  <EmployeeViewWrapper>
+                    <EmployeeView employee={methods.getValues()}></EmployeeView>
+                  </EmployeeViewWrapper>
+                </Fieldset>
+                <ButtonGrpWrapper>
+                  <Button icon="" onClick={() => setActiveSection(1)}>
```

Comment:
use edit button icon instead of this edit button on the top right of the page.. near the review